build/Tests/UnitTests/Foundation.WindowsOnly/Foundation.WindowsOnly.UnitTests.vcxproj

```diff
@@ -234,8 +234,6 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\Frameworks\Foundation\NSObject_NSCoding.mm" />
```

Comment:
>langCompile Include="$(StarboardBasePath)\Frameworks\Foundation\NSObject_NSCoding.mm" /> [](start = 6, length = 88)

out-of-scope; This should be unnecessary since the `/WHOLEARCHIVE` fixes